### PR TITLE
core(password-inputs-can-be-pasted-into): add devtoolsNodePath

### PR DESCRIPTION
**Pull request by adrianaixba**
small addition to pr #11405 

addresses "audits that use nodes but not devtools node paths" for password-inputs-can-be-pasted-into #9947

### CODE REVIEW

File: lighthouse-core/audits/accessibility/axe-audit.js
Comment (adrianaixba) [inline lighthouse-core/audits/accessibility/axe-audit.js:82]: Not 100% about this. I assumed that NodeValue selector should be the actual node's selector. But, this originally uses the aXe audit's css selectors (given by `node.target`), which might just be an audit exception?

File: lighthouse-core/audits/dobetterweb/password-inputs-can-be-pasted-into.js
Comment (connorjclark) [inline lighthouse-core/audits/dobetterweb/password-inputs-can-be-pasted-into.js:48]: I don't think this type comment will work: you must wrap the object value in parentheses, and put the comment just before it (look around for similar usages)

you can confirm it may not work by change 'node' to some random gibberish and note that there may not be a type error on the object

File: lighthouse-core/audits/accessibility/axe-audit.js
Comment (adamraine) [reply lighthouse-core/audits/accessibility/axe-audit.js:82]: I think it's ok to keep this the way it is. @connorjclark are we worried about the difference between the selector returned by aXe and the selector we construct in `getNodeSelector()`?

File: lighthouse-core/audits/accessibility/axe-audit.js
Comment (adamraine) [inline lighthouse-core/audits/accessibility/axe-audit.js:81]:
```suggestion
          // This selector represents the aXe CSS selectors
```

File: lighthouse-core/test/results/sample_v2.json
Comment (adamraine) [inline lighthouse-core/test/results/sample_v2.json:3540]: Looks like some snippets are not being returned as well

**Review by adamraine — COMMENTED** [review]
@adrianaixba Are you planning to merge this PR before or after you merge https://github.com/GoogleChrome/lighthouse/pull/11405?

File: lighthouse-core/test/results/sample_v2.json
Comment (adrianaixba) [reply lighthouse-core/test/results/sample_v2.json:3540]: planning on merging this after #11405! these sample_v2 errors are coming from there, should be fixed over there soon

File: lighthouse-core/audits/accessibility/axe-audit.js
Comment (connorjclark) [reply lighthouse-core/audits/accessibility/axe-audit.js:82]: @paulirish didn't you look at the difference here between axe and us?

Comment (paulirish) [reply lighthouse-core/audits/accessibility/axe-audit.js:82]: @adrianaixba and I just deep dived into .target and what it's useful for. It addresses cases where the node is within shadowDOM or iframes. (because css selectors can't express that.)

Since Lighthouse can't do anything useful here (and our own getSelector doesnt handle the iframe/shadow DOM cases) I think we should ignore it.

So yeah let's use our own NodeValue selector here and discard their `target` selector array.

File: lighthouse-core/audits/accessibility/axe-audit.js
Comment (adrianaixba) [reply lighthouse-core/audits/accessibility/axe-audit.js:81]: Since we're ignoring aXe selector in this case, the comment is not necessary anymore.

**Comment by adamraine** [comment]
@adrianaixba is this waiting on me?

File: lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js
Comment (connorjclark) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js:77]: can we update the PR title to reflect these changes too?

if can't fix, can mention in the body.

**Review by connorjclark — APPROVED** [review]

File: lighthouse-core/test/results/artifacts/artifacts.json
Comment (adrianaixba) [inline lighthouse-core/test/results/artifacts/artifacts.json:1294]: @connorjclark I talked to @patrickhulce offline about this. After changing the axe-audit selector, there were changes  for the artifacts. We noticed that there hasn't been an update for axe in some time. That's why there's a lot of diff on this file, so would need another review from you, what do you think?

Comment (connorjclark) [reply lighthouse-core/test/results/artifacts/artifacts.json:1294]: LGTM

**Pull request merged**
